app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/state/MessageComposerStateHolder.kt

```diff
@@ -77,6 +68,23 @@ fun rememberMessageComposerStateHolder(
             )
         )
     }
+
+    LaunchedEffect(draftMessageComposition.draftText) {
+        if (draftMessageComposition.draftText.isNotBlank()) {
+            messageTextFieldValue.value = messageTextFieldValue.value.copy(
+                text = draftMessageComposition.draftText,
+                selection = TextRange(draftMessageComposition.draftText.length) // Place cursor at the end of the new text
+            )
+        }
+
+        if (draftMessageComposition.selectedMentions.isNotEmpty()) {
```

Comment:
question: shouldn't that have its own `LaunchedEffect` to be executed every time when `draftMessageComposition.selectedMentions` changes? 🤔

Reply:
that is not necessary so far, as `selectedMentions` is updated only with `draftText` (in `MessageDraftViewModel.loadMessageDraft()`)

Reply:
that is not necessary so far, as `selectedMentions` is updated **only** with `draftText` (in `MessageDraftViewModel.loadMessageDraft()`)